hexa/workspaces/schema.py

```diff
@@ -73,6 +74,11 @@ def resolve_workspace_permissions(workspace: Workspace, info):
     return workspace
 
 
+@workspace_object.field("database")
+def resolve_workspace_database(workspace: Workspace, info, **kwargs):
+    return Database.objects.get(database="hexa-explore-demo")
```

Comment:
We should hide the implementation details of this first version and not return the database but the workspace instead.
```suggestion
    return workspace
```